playbooks/library/gather_disk_device_facts

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/env python
+# Copyright 2018, Rackspace US, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import shlex
+import subprocess
+import json
+
+from ansible.module_utils.basic import *  # noqa: ignore=H303
+
+
+DOCUMENTATION = """
+---
+module: gather_disk_device_facts
+short_description:
+    - A module for gathering disk device facts.
+description:
+    - A module for gathering disk device facts.
+author: Rcbops
+"""
+
+EXAMPLES = """
+- name: Gather ceph osd host facts
+  ceph_osd_host_facts:
+"""
+
+
+class DiskDeviceFacts(object):
+    def __init__(self, module):
+        self.state_change = False
+        self.module = module
+
+    def gather_facts(self, hostname, container_name):
+        """Get disk device facts on a host"""
+        # in case we need to gather disks inside of a container
+        _ = container_name
+        _ = _ = hostname
+        disk_device_command_string = ""
```

Comment:
If Im reading this right, this is an empty command so it'd return nothing? Is this supposed to be user configurable, or is that coming soon?